### PR TITLE
Use enueration for policy items

### PR DESCRIPTION
**Pull request by porridge**

### CODE REVIEW

**Review by sulenn — APPROVED** [review]

**Review by sulenn — APPROVED** [review]

File: ISSUES.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
-If you open a GitHub Issue, here is our policy: 1. It must be a bug/performance
-issue or a feature request or a build issue or a documentation issue (for small
-doc fixes please send a PR instead). 2. Make sure the Issue Template is filled
-out. 3. The issue should be related to the repo it is created in.
+If you open a GitHub Issue, here is our policy:
+1. It must be a bug/performance issue or a feature request or a build issue or a documentation issue (for small doc fixes please send a PR instead).
```
Comment (mihaimaruseac) [inline ISSUES.md:2]:
```suggestion
If you open a GitHub Issue, here is our policy:

1. It must be a bug/performance issue or a feature request or a build issue or a documentation issue (for small doc fixes please send a PR instead).
```

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]

**Review by mihaimaruseac — APPROVED** [review]

**Comment by mihaimaruseac** [comment]
@porridge it seems you have changed your email address on GitHub/commits? CLA bot says you have to sign CLA again.

**Comment by gbaned** [comment]
@porridge  Any update on this PR? Please. Thanks!

**Comment by porridge** [comment]
@googlebot I signed it!

**Review by MarkDaoust — APPROVED** [review]

**Pull request merged**